core/token-pooling/token-pool.js

```diff
@@ -80,6 +80,10 @@ class Token {
     return this.usesRemaining <= 0 && !this.hasReset
   }
 
+  get decrementedUsesRemaining() {
+    return this._usesRemaining - 1
+  }
+
```

Comment:
The rationale/necessity for this is explained in the inline comments in the `getRateLimitFromHeaders` function in the new `librariesio-api-provider` class